objects/scienceoutpost/scienceinfobooth/scienceinfobooth.object

```diff
@@ -0,0 +1,91 @@
+	"Some of the information in those codexes might not seem relevant right now, but even some of our veteran researchers have learned something they didn't know through those books, so keep them in mind!",
+	"Looks like most of your Protectorate crafting databanks have been scrambled at some point - You'll need to use your Tricorder to research how to craft a lot of things.",
+	"I'd recommend researching extraction technology sooner rather than later. A Hand Mill might not be as fast as a full Extraction Lab, but it's still one of the most powerful tools in your arsenal.",
+    "Having trouble keeping your ship's tank topped up? Remember, Erchius isn't the only fuel you can use - Radioactive isotopes, properly processed algae, and even pure Hydrogen can all act as fuel sources - and some are more effective than others!",
+    "Having trouble gaining research? Your passive research gain will increase the longer you play, and will also increase based on the tier of the planet you're on - Exploring more dangerous planets lets you gather research more quickly!",
+    "Okay, so a couple of coworkers were saying they were leaving to go snarsh some garflipps, and I asked them what that meant, and they looked at me like I was an idiot. Is that normal? Am I the wierd one for not knowing what that means?",
+	"A randomly-generated weapon will almost always be weaker than one you've crafted yourself. Crafted weapons also generally have special abilities designed to complement the weapon. Consider using a computer to turn randomly-generated weapons into research, instead!",
+    "Have a favorite weapon or armor set that you don't want to leave behind as you progress? Good news - you don't have to! Using your Tricorder's Upgrade function, you can use Upgrade modules to improve almost all Frackin' Universe weapons or armor. By doing this, you can ensure that any armor or weapon you like stays viable for the entire game!",
+    "Does an enemy seem almost invincible? It might just be resistant to the damage type you're using. Try switching things up with different weapons to find a weakness!",
+    "Don't know where to find a specific item or material? A Lab Directory can tell you if it's an output from any of your nearby machines. If that fails, check the official wiki displayed on the main menu!",
+    "Even carnivores have a reason to set up plant farms - many of the growable plants that Frackin' Universe adds can be eaten by anyone. Plus, Meatvine is a reliable source of nutrition, and many plants and their seeds can be extracted into useful materials!",
+    "Sorry we can't let you use most of the crafting and processing machines - We had to restrict access after Kevin tried to use them to create a gun that shot smaller versions of itself.",
```

Comment:
"Gun that shoots smaller versions of itself" might as well be a placeholder - I just thought it'd be a funny explanation for why the player can't use the Science Outpost machines. If there was more room, I'd have the person at the info booth remark that they're not sure _why_ limiting access worked, since Kevin could probably bypass it, but for some reason he doesn't. I dunno. I just like the idea that they've found something that keeps Kevin in check, but logically it _shouldn't,_ and they have no clue why it works.